packages/chatbot-server-mongodb-public/src/processors/findContentWithMongoDbMetadata.test.ts

```diff
@@ -0,0 +1,92 @@
+// Mocks
+jest.mock("mongodb-rag-core/mongoDbMetadata", () => {
+  const actual = jest.requireActual("mongodb-rag-core/mongoDbMetadata");
+  return {
+    ...actual,
+    classifyMongoDbProgrammingLanguageAndProduct: jest.fn(),
+  };
+});
+
+jest.mock("mongodb-rag-core", () => {
+  const actual = jest.requireActual("mongodb-rag-core");
+  return {
+    ...actual,
+    updateFrontMatter: jest.fn(),
+  };
+});
+
+jest.mock("mongodb-rag-core/braintrust", () => {
+  const actual = jest.requireActual("mongodb-rag-core/braintrust");
+  return {
+    ...actual,
+    wrapTraced: jest.fn(),
+  };
+});
```

Comment:
are you sure that you need to mock this?

Reply:
You're right, I don't. Removed.